site/_posts/2016-12-12-last-week-in-manageiq-provider-generator-pglogical-chen-primes.html.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Last Week in ManageIQ: Provider generator, pglogical upgrades, and Chen primes!"
+author: chrisarcand
+date: 2016-12-13 14:05:10 -06:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Chris](https://twitter.com/chrisarcand) here, welcoming you to another edition
+of *Last Week in ManageIQ*! It's a balmy 0° here in Minnesota, USA - that's
+about -18° Celsius, according to local authorities. With the wind chill making
+it -20° (-29°C), I'm thinking a look back at all the great work done on
+ManageIQ for the past week is just what we need to keep warm on this fine
+winter day!
+
+## Featured
+
+There were [107 pull requests merged to master][PRs merged last week] last week! 107 is the 28th prime number.
+The next prime is 109, meaning that 107 is called a [Chen prime](https://en.wikipedia.org/wiki/Chen_prime).
+Super cool, right? Almost as super cool as 0°! If I haven't scared you off yet,
+you can join the 45 contributors that made [255 commits][Commits merged last week] by
```

Comment:
Ugh it's because commits that are merged AFTER the cutoff but committed (written) BEFORE the cutoff are now counted in that calculation. So this number will likely be inaccurate again soon.

I wonder if we can be more accurate through some git magic of our own sometime cc/ @imtayadeway